dataprep/clean/clean_date.py

```diff
@@ -0,0 +1,996 @@
+                else:
+                    result = result.replace(year_token, str(year))
+    return result
+
+
+def transform_month(
+    result_str: Union[str, Any], month_token: Union[str, Any], month: Union[int, Any]
+) -> Any:
+    """
+    This function transform parsed month into target format
+    Parameters
+    ----------
+    result_str
+        result string
+    month_token
+        token of month
+    month
+        value of month
+    """
+    result = deepcopy(result_str)
+    if month_token != "":
+        if month == -1:
+            if len(month_token) == 3:
+                result = result.replace(month_token, str("---"))
+            elif len(month_token) == 5:
+                result = result.replace(month_token, str("-----"))
+            elif len(month_token) == 2:
+                result = result.replace(month_token, str("--"))
+            elif len(month_token) == 1:
+                result = result.replace(month_token, str("-"))
+        else:
+            if len(month_token) == 2:
+                if month < 10:
+                    result = result.replace(month_token, "0" + str(month), 1)
+                else:
+                    result = result.replace(month_token, str(month), 1)
+            elif len(month_token) == 3:
+                result = result.replace(month_token, TEXT_MONTHS[month - 1][0], 1)
+            elif len(month_token) == 5:
+                result = result.replace(month_token, TEXT_MONTHS[month - 1][1], 1)
+            else:
+                result = result.replace(month_token, str(month), 1)
+    return result
+
+
+def transform_day(
+    result_str: Union[str, Any], day_token: Union[str, Any], day: Union[int, Any]
+) -> Any:
+    """
+    This function transform parsed month into target format
+    Parameters
+    ----------
+    result_str
+        result string
+    day_token
+        token of day
+    day
+        value of day
+    """
+    result = deepcopy(result_str)
+    if day_token != "":
+        if day == -1:
+            if len(day_token) == 2:
+                result = result.replace(day_token, str("--"))
+            elif len(day_token) == 1:
+                result = result.replace(day_token, str("-"))
+        else:
+            if len(day_token) == 2:
+                if day < 10:
+                    result = result.replace(day_token, "0" + str(day), 1)
+                else:
+                    result = result.replace(day_token, str(day), 1)
+            else:
+                result = result.replace(day_token, str(day))
+    return result
+
+
+def transform_hms(
+    result_str: Union[str, Any],
+    hms_token: Union[str, Any],
+    ispm: Union[bool, Any],
+    hms_value: Union[int, Any],
+) -> Any:
+    """
+    This function transform parsed month into target format
+    Parameters
+    ----------
+    result_str
+        result string
+    hour_token
+        token of hour
+    hour
+        value of hour
```

Comment:
The comment is not consistent with the parameters

Reply:
Refined.